docs/features/banner-designer.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Banner Designer"
+description: "Customize the consent banner & preferences modal to match your branding."
```

Comment:
```suggestion
description: "Customise the consent banner & preferences modal to match your branding."
```

Reply:
we want to use EU spelling since most of our clients are EU and there are more regulations here anyway.